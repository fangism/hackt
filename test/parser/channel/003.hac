// "003.hac" should fail, need data-type in list

defchan mychan <: chan(pbool) (int i);

