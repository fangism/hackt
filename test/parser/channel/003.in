// "001.in" should fail, need data-type in list

defchan mychan <: chan(pbool) (int i);

