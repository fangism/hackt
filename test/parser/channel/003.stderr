parse error: syntax error
parser stacks:
state	value
#STATE#	(null) 
#STATE#	(null) 
#STATE#	keyword: defchan [3:1..7]
#STATE#	identifier: mychan [3:9..14]
#STATE#	<: [3:16]
#STATE#	(chan-type) [3:19..22]
#STATE#	( [3:23]
#STATE#	paramtype: pbool [3:24..28]
in state #STATE#, possible rules are:
	data_type_ref_list_in_parens : '(' . data_type_ref_list ')'  (#RULE#)
acceptable tokens are: 
	ID (shift)
	SCOPE (shift)
	INT_TYPE (shift)
	BOOL_TYPE (shift)
